src/utils/getLiquidityUrlPathParts.ts

```diff
@@ -1,9 +1,15 @@
 // Constructing the two forward-slash-separated parts of the 'Add Liquidity' URL
 // Each part of the url represents a different side of the LP pair.
-const getLiquidityUrlPathParts = ({ quoteTokenAdresses, tokenAddresses }) => {
+import { getWbnbAddress } from './addressHelpers'
+
+const getLiquidityUrlPathParts = ({ quoteTokenAddress, tokenAddress }) => {
   const chainId = process.env.REACT_APP_CHAIN_ID
-  const firstPart = quoteTokenAdresses ? quoteTokenAdresses[chainId] : 'BNB'
-  const secondPart = tokenAddresses ? tokenAddresses[chainId] : 'BNB'
+  const wBNBAddressString = getWbnbAddress()
```

Comment:
can you explain why this change is needed ?

Reply:
Because farm quote token contains wbnb address instead of null, that causes we still redirect user to (wbnb - token ) add liquidity page.